timed/permissions.py

```diff
@@ -141,3 +168,150 @@ class IsNotBilledAndVerfied(BasePermission):
 
     def has_object_permission(self, request, view, obj):
         return not obj.billed or obj.verified_by_id is None
+
+
+class IsInternal(IsAuthenticated):
+    """Allows access only to internal employees."""
+
+    def has_permission(self, request, view):
+        if not super().has_permission(request, view):  # pragma: no cover
+            return False
+
+        if request.method not in SAFE_METHODS:
```

Comment:
Drop the request method check. This permission should only check if the user is internal, not anything else. You can combine the permission with other permissions to achieve the actual check you need